lib/index.ts

```diff
@@ -57,7 +57,11 @@ export const storyblokInit = (options: SbSvelteSDKOptions) => {
 };
 
 export const getComponent = (componentName: string) => {
-  const component = componentsMap[componentName];
+  let component = null;
+  component =
```

Comment:
Calling the callback function on every `getComponent` run can lead to performance issues, especially with large component maps. I would run it just once and keep the result on `componentsMap`